velox/exec/RowContainer.cpp

```diff
@@ -567,7 +562,8 @@ void RowContainer::store(
         decoded,
         rows,
         isKey,
-        offsets_[column]);
+        offsets_[column],
+        column);
```

Comment:
This is also change in https://github.com/facebookincubator/velox/pull/11527, FYI.
Thanks.